docs/content/start/telemetry.md

```diff
@@ -0,0 +1,175 @@
+    TelemetryLogger.prepareErrorObject(newEvent, error, false);
+  }
+  this.send(newEvent);
+}
+```
+
+However, the code snippet here shows a telemetry event object that adds much more complex logics to determine the event that has occurred, then passed the resulting event to the `send()` method for the container's telemetry system to output.
+
+Here is the above telemetry event object in action:
+
+```ts
+// Adding this event temporarily so that we can get help debugging if something goes wrong.
+this.logger.sendTelemetryEvent({
+  eventName: "connectedStateRejected",
+  source,
+  pendingClientId: this.pendingClientId,
+  clientId: this.clientId,
+  hasTimer: this.prevClientLeftTimer.hasTimer,
+  inQuorum: protocolHandler !== undefined && this.pendingClientId !== undefined
+      && protocolHandler.quorum.getMember(this.pendingClientId) !== undefined,
+});
+```
+
+## Code example
+
+With the interface already hooked up to the container's telemetry system, it is easy for users to write a custom telemetry object by implementing the `ITelemetryBaseLogger` interface and defining the `send()` method. Below is an example custom telemetry logger, `ConsoleLogger`, that implements the `ITelemetryBaseLogger` interface. As the name suggest, the `ConsoleLogger` defined the `send()` method to stringify the entire event object and print it out.
+
+```ts
+import { ITelemetryBaseLogger, ITelemetryBaseEvent } from "@fluidframework/common-definitions";
+
+// Define a custom ITelemetry Logger. This logger will be passed into TinyliciousClient
+// and gets hooked up to the Tinylicious container telemetry system.
+export class ConsoleLogger implements ITelemetryBaseLogger {
+    constructor() {}
+    send(event: ITelemetryBaseEvent) {
+        console.log("Custom telemetry object array: ".concat(JSON.stringify(event)));
+    }
+}
+```
+
+This custom logger is then created and passed into the `getContainer()` and `createContainer()` in a `TinyliciousClient` object. The custom logger is now hooked up to the container's telemetry system.
+
+```ts
+async function start(): Promise<void> {
+    // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
+    // and hook to the Telemetry system
+    const consoleLogger: ConsoleLogger = new ConsoleLogger();
+
+    // Get or create the document depending if we are running through the create new flow
+    const client = useFrs ? FrsClient :  new TinyliciousClient();
+    const [fluidContainer, containerServices] = createNew
+        ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
+        : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
+```
+
+Now, whenever a telemetry event is encountered, the custom `send()` method gets called and will print out the entire event object.
+
+![ConsoleLogger_telemetry_in_action](/images/ConsoleLogger_telemetry_in_action.png "ConsoleLogger_telemetry_in_action")
```

Comment:
the image isn't showing up on Github, but so are other images in other markdowns.